webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,40 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+/* eslint-disable max-params  */
+
+export function notify(message, type, link, sticky) {
+  showToast({
+    message,
+    type,
+    sticky,
+    link
+  });
+}
+
+function showToast(toast) {
```

Comment:
Is there any reason not to unify these two functions? all this function does is pass it's parameter to another function...